Lombiq.Tests.UI/Constants/DirectoryPaths.cs

```diff
@@ -9,8 +10,8 @@ public static class DirectoryPaths
     public const string Temp = nameof(Temp);
     public const string Screenshots = nameof(Screenshots);
 
-    public static string GetTempSubDirectoryPath(string contextId, string subDirectoryName) =>
-        Path.Combine(Environment.CurrentDirectory, Temp, contextId, subDirectoryName);
+    public static string GetTempSubDirectoryPath(string contextId, params string[] subDirectoryNames) =>
+        Path.Combine(new[] { Environment.CurrentDirectory, Temp, contextId }.Union(subDirectoryNames).ToArray());
```

Comment:
`.Union()` is a set operator. Only use it if you want to treat your collection as a set. In this case use `.Concat()`.
![image](https://user-images.githubusercontent.com/4007293/162466327-521cdae0-52b2-4c97-b665-9cada461700d.png)

Reply:
Right.